api/app/assets/javascripts/mno_enterprise/config.js.coffee.erb

```diff
@@ -1,9 +1,15 @@
 # Configuration module
 #
-# Recompile if the settings change
+# Dist: Recompile front-end if the settings change
+# Dev: Delete express project public/assets to ensure nothing is cached, bundle and restart.
 # = depend_on_config
 angular.module('mnoEnterprise.configuration', [])
-  .constant('IMPAC_CONFIG', <%= Settings.impac.to_json %>)
+  .constant('IMPAC_CONFIG', {
+    settings: <%= Settings.impac.to_json %>,
+    keys: {
+      pusher_client: <%= ((k = ENV['pusher_key']) && k.to_json) || ''  %>
```

Comment:
@ouranos maybe you can think of a better way I can do this? I want the value to be falsey should `env['pusher_key']` be nil.